packages/helm-charts/celo-fullnode/rc1-blockscout-archive-nodes.yaml

```diff
@@ -0,0 +1,148 @@
+  - 8408dcd47dcb987c4c89033f7b8506901c66e181a10458ebcc91eeef982c2582
+  - 202b60fdb9353776f8f4e00225ff02115d666540a56cf549d30818e299cd4d59
+  - 8d29fa939f8941c49b4f07b0ad0a97c6eb239e5880692a072d4e1be2c463b4b0
+  - 9f803e572b044fdfae9c7f3e74dee7c2162d233ba423693334008125a648b274
+  - 0386a0924d55c55e7ede22f2e9e97d6d35f5701131d8731fd0da151126806ffd
+  - 0b2fd5138854399a8219b0dd1784f244b9f9fc296d181d298a32867187b16a67
+  - 0182756023478f62374dd687afdccfd51851486377accc96f4a4c1db25744f99
+  - 3fe6b563d9716770e0299118fca2ba1ccc7614790d34ab2edb1641a5746985df
+  - f2a08b997813997b0bdda27fba307febf289d74b447fc2249034fe871a6f49c7
+  - 22c0c78eb636490dd207314752e1688c54904d1f0f30fdc3a4def0546e4901ed
+  - f1c92539640b122db86c32bf746174bcaa54c9ee0824ea68aef1d625721b3f68
+  - b8674e3f79f79790209c4e75040e31ce321184085c8d49b71341a45c86b050bc
+  - f2939dbec209f51467e406a4e80884e074c533bafe378cced62a19f14f454e73
+  - 8bad5dbd251fd52499ded9ce5e4944dbd1f9d6208546537763998051c49f8f91
+  - 71d8c4e8ac3611f208e0f01391738d3f6146be3f1f535aae0d54aa66de83caad
+  - 069163e28ee56776b26d354831586422ca84e01c500f158709b413c76fb2a68e
+  - 1c83157e1c8dd52ea9f826c15acc8c8a4de3338746ab411386879135440b3085
+  - 5eba03f3b36e92b711d58b6fa9a66c8342c86cabe11ab748f4a66fa504c3e9ff
+  - 04851878b38b9509de08f7c02bc49f949937cd5dabb6aea71c552f4f86734d1f
+  - 00c3206352cb7b855ed72675e62192f07cef72717b8cacf1add709d46acd7a89
+  - c37112d3e511aff3113512b46a1433ebe473c4bbf55aeca33c321c64bf075dd8
+  - 9584d2a83be292abd8de0332af9c5fc327e98ee4f190340f30a9cd7a65ae606d
+  - cd81f5f300e8fe070e7c5b867a1c2a531f5d4c3d1b7a1b1b49258c6e4289be13
+  - 82bfe2e82c3ee2331cbbe14ca03a286b19c691785d7db8d4213919b4a970e5c4
+  - 08508d726d3d592e922324eacdcf622480a479c61a0414e47b59e219b7c3c245
+  - 0176d2e8d1a3fc60f2cc55265ebe2bfb04074521d52b1fb844e2fe28646f604b
+  - 53cc64a704ac5712d1990efd511571d4b22a566c1c184731dda57c04acc815c9
+  ping_ip_from_packet: false
+  public_ip_per_node:
+  - 34.168.166.173
+  - 34.168.53.31
+  - 34.168.142.153
+  - 104.196.248.95
+  - 34.168.14.34
+  - 35.199.167.145
+  - 34.168.58.236
+  - 34.168.208.61
+  - 35.230.20.151
+  - 35.203.178.255
+  readiness_probe_max_block_age_seconds: 30
+  readiness_probe_max_block_epoch_age_seconds: 300
+  resources:
+    requests:
+      cpu: "7"
+      memory: 24Gi
+  rpc_apis: eth,net,rpc,web3
+  service_protocols:
+  - TCP
+  - UDP
+  service_type: LoadBalancer
+  syncmode: full
+  updateStrategy:
+    rollingUpdate:
+      partition: 0
+    type: RollingUpdate
+  use_gstorage_data: false
+  use_static_ips: true
+  verbosity: 2
+  ws_port: 8545
+ingress:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    konghq.com/plugins: forno-rate-limit, forno-auth
+    konghq.com/strip-path: "true"
+  enabled: false
+  ingressClassName: nginx
+metrics: true
+namespace: blockscout
+nodeSelector:
+  pool: t2d-standard-8-spot
+pprof:
+  enabled: true
+  path: /debug/metrics/prometheus
+  port: 6060
+prometheus: true
+replicaCount: 1
```

Comment:
should this initially be higher?
or is the intention to increase it once the other archive node pool is scaled down?

Reply:
Once it is used we may have a different initial count, although this value will be affected by max and min replicas from hpa.